include/glow/Quantization/Base/Base.h

```diff
@@ -69,7 +69,7 @@ enum Schema {
   /// (i.e., offset == -128).
   /// An offset of -128 represents an unsigned int8 with an offset of zero:
   /// int8 is [-128; 127] - (-128) == uint8 [0; 255] - 0
-  SymmetricWithUInt8,
+  SymmetricWithUnsigned,
```

Comment:
could you update doxygen comments for `SymmetricWithUnsigned`

Reply:
Done.